generic3g/tests/Test_FixedLevelsVerticalGrid.pf

```diff
@@ -13,9 +13,22 @@ contains
 
       real, parameter :: levels(*) = [1.,5.,7.]
 
-      vgrid = FixedLevelsVerticalGrid(standard_name='air_pressure', levels=levels)
+      vgrid = FixedLevelsVerticalGrid(standard_name='air_pressure', units='Pa', levels=levels)
```

Comment:
In theory the units can come from the FIeldDictionary, but this is fine for now.

Reply:
@tclune Not in this case, the primary use for the will be History, the collection will define the vertical coordinate, up to the user what units they want to specify the levels in that they are going to type in their History file